yt_dlp/extractor/megatube.py

```diff
@@ -0,0 +1,40 @@
+from .common import InfoExtractor
+from ..utils import js_to_json
+
+
+class MegaTubeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?megatube\.xxx/videos/(?P<id>\d+)'
+
+    _TESTS = [{
+        'url': 'https://www.megatube.xxx/videos/104245/brunette-abbie-cat-with-big-fake-tits-gets-the-fuck-she-wants/',
+        'md5': '8df46b28ce6d7ea00cb4d09adf881cd3',
+        'info_dict': {
+            'id': '104245',
+            'ext': 'mp4',
+            'title': 'Brunette Abbie Cat with big fake tits gets the fuck she wants',
+            'thumbnail': 'https://www.megatube.xxx/contents/videos_sources/104000/104245/screenshots/9.jpg',
+            'age_limit': 18,
+        }
+    }, {
+        'url': 'https://www.megatube.xxx/videos/104245/brunette-abbie-cat-with-big-fake-tits-gets-the-fuck-she-wants/',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        data_json = self._search_json(
+            r'var\s?flashvars\s*=', webpage, 'data', video_id,
+            transform_source=js_to_json)
+
+        title = (self._og_search_title(webpage, default=None)
+                 or self._html_extract_title(webpage).split('(')[0]).strip()
```

Comment:
Why can't title contain `(`?

Reply:
On this particular website, the title of the page contains an extra suffix in parenthesis.

Reply:
Yes, but the title can contain other `(` too, no?

Reply:
might be a job for `utils.remove_end()`